### PR TITLE
feat: shifted from yarn to npm

### LINKED ISSUE #220: feat: unify the package managers used in examples
We need to unify the package managers used in this project, but it is not related to this PR. I think `npm` is a good choice because it's general.

_Originally posted by @yusukebe in https://github.com/honojs/examples/pull/219#discussion_r1858601057_

### PR DESCRIPTION
**Pull request by MathurAditya724**
Closed #220 

- [x] Shifted from `yarn` to `npm` workspace
- [x] Updated the README's of the projects

### CODE REVIEW

**Comment by yusukebe** [comment]
Hi @MathurAditya724 

Thanks!

I've grepped, but there are still `yarn`s. Should you update them?

CleanShot 2024-12-06 at 16 21 05@2x

**Comment by MathurAditya724** [comment]
I must have missed those, I have updated the repo. Kindly have a look

File: basic/README.md [hunk 1]
```diff
@@ -5,7 +5,7 @@ Before you can start publishing your service to cloudflare worker, you must sign
 You can update the information (`name`, `zoon_id`, etc) in wrangler file, then you can test and deploy your service by simply doing,
 
 ```txt
-yarn install
-yarn dev 		# Start a local server for developing your worker
-yarn deploy # Publish your worker to the orange cloud
+npm install
+npm dev # Start a local server for developing your worker
```
Comment (yusukebe) [inline basic/README.md:9]: This should be `npm run dev`. The others also should include `run`.

File: basic/README.md [hunk 2]
```diff
@@ -5,7 +5,7 @@ Before you can start publishing your service to cloudflare worker, you must sign
 You can update the information (`name`, `zoon_id`, etc) in wrangler file, then you can test and deploy your service by simply doing,
 
 ```txt
-yarn install
-yarn dev 		# Start a local server for developing your worker
-yarn deploy # Publish your worker to the orange cloud
+npm install
+npm run dev # Start a local server for developing your worker
+npm deploy # Publish your worker to the orange cloud
```
Comment (yusukebe) [inline basic/README.md:10]: `npm run deploy`.

File: blog/README.md [hunk 3]
```diff
@@ -63,11 +63,11 @@ kv_namespaces = [
...
 ```sh
-yarn dev
+npm run dev
 ```
 
 Publish:
 
 ```sh
-yarn deploy
+npm deploy
```
Comment (yusukebe) [inline blog/README.md:72]: `npm run deploy`.

File: pages-stack/package.json [hunk 4]
```diff
@@ -3,9 +3,9 @@
   "version": "0.0.0",
   "type": "module",
   "scripts": {
-    "dev": "wrangler pages dev --compatibility-date=2023-02-05 -- yarn vite",
+    "dev": "wrangler pages dev --compatibility-date=2023-02-05 -- npm vite",
     "build": "vite build",
-    "deploy": "yarn build && wrangler pages deploy pages"
+    "deploy": "npm build && wrangler pages deploy pages"
```
Comment (yusukebe) [inline pages-stack/package.json:8]: `"deploy": "npm run build && wrangler pages deploy pages"`

**Review by yusukebe — APPROVED** [review]
LGTM!

**Pull request merged**